packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -130,6 +135,47 @@ public ResponseEntity<List<Project>> getProjects(
 		return ResponseEntity.ok(projects);
 	}
 
+	private class Contributor {
+		String name;
+		Schema.Relationship permission;
+
+		Contributor(String name, Schema.Relationship permission) {
+			this.name = name;
+			this.permission = permission;
+		}
+	}
+
+	private List<Contributor> getContributors(RebacProject rebacProject) {
+		Map<String, Contributor> contributorMap = new HashMap<>();
+
+		try {
+			List<RebacPermissionRelationship> permissionRelationships = rebacProject.getPermissionRelationships();
+			for (RebacPermissionRelationship permissionRelationship : permissionRelationships) {
+				Schema.Relationship relationship = permissionRelationship.getRelationship();
+				if (relationship.equals(Schema.Relationship.CREATOR)
+						|| relationship.equals(Schema.Relationship.ADMIN)
+						|| relationship.equals(Schema.Relationship.WRITER)) {
+					if (permissionRelationship.getSubjectType().equals(Schema.Type.USER)) {
+						PermissionUser user = reBACService.getUser(permissionRelationship.getSubjectId());
+						String name = user.getFirstName() + " " + user.getLastName();
```

Comment:
Should we add a method in `PermissionUser` to return the fullname instead?

Reply:
Great question.  There is seems to be a function required here and in the front end for this feature.  Ideally this is a task of its own to capture this need and make it system-wide.